guides/Version_Control/GitHubSSH.md

```diff
@@ -0,0 +1,101 @@
+# GitHub Secure Shell
+
+#### What is SSH?
+
+_Secure Shell (SSH) is a cryptographic network protocol for operating network services securely over an unsecured network._ [wikipedia][1]
+
+**In a nutshell, SSH protocol allows connecting to and authenticating remote servers and services. This protocol requires ssh keys which facilitates connecting to Github without providing username and password credentials.**
+
+
+#### Do I have existing SSH keys?
+It is always good practice to look before you leap.
+
+##### To check:
+At the command line type the following:
+```
+    $ ls -al ~/.ssh
+```
+Note: **DO NOT ENTER**  __*$*__  it simply signifies that you are at the command line.  
+
+The above lists the files in a directory named  **.ssh**, if they exist. The following are default file names:
+
+> id_dsa.pub  
+ id_ecdsa.pub  
+id_ed25519.pub  
+id_rsa.pub  
+
+#### Keys to the kingdom
+1. Execute this command adding your Github **email address** to **create a key pair**:  
+
+  `$ ssh-keygen -t rsa -b 4096 -C "your_email@example.com"
+  `  
+
+2. Press **Enter** to save the file to the default location when prompted. You have the option to save the file elsewhere.  
+
+3. Type in a **passphrase** or **opt out** by leaving it empty when prompted.
+
+4. Confirm you **passphrase**.  
+
+#### Add keys via ssh-agent
+1. ```
+  $ eval "$(ssh-agent -s)"
```

Comment:
This is not highlighted text. ```1.``` appears at top while  ```$ eval "$(ssh-agent -s)"``` appears down. Kindly fix this.